<launch>
  <!-- launch plane segment process -->
  <include file="$(find pcl_something)/launch/plane_segment.launch"/>

  <node name="Euclid_Cluster_Extraction" pkg="pcl_something" type="euclidean_clustering">
    <param name="tolerance" value="0.05"/>
    <param name="min_cluster" value="100"/>
    <param name="max_cluster" value="10000"/>
  </node>
  
</launch>